ci-operator/step-registry/cucushift/installer/check/ibmcloud/destroy/cucushift-installer-check-ibmcloud-destroy-commands.sh

```diff
@@ -27,8 +27,10 @@ ibmcloud_login
 CLUSTER_FILTER="${NAMESPACE}-${UNIQUE_HASH}"
 hasError=0
 
+# Disable exit on error to allow for no resource groups remains.
+set +e
```

Comment:
If you do not want the behavior of `exit on error`, you can directly comment out the line 4 globally, the exit value would depend on your control of `hasError` in this script.

Reply:
just the grep command will cause the exit, if the resource group has been deleted, grep will cause an error exit. 
And I want to use global errexit to control other unexpected errors.

Reply:
My suggestion is if you want to use global errexit, you should try best to update the command line to avoid the unexpected exit but not `set +e` to escape it. If next time you found other place also lead some unexpected exit, then still use `set +e` to escape it? If that, the global make no much sense. 

And more, even turn off  global errexit, from the code, I did not see much risk.